migrations/versions/14e31b3a860b_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: 14e31b3a860b
+Revises: 09c86e38f5e7
+Create Date: 2019-06-15 23:52:13.462368
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '14e31b3a860b'
+down_revision = '09c86e38f5e7'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events', 'is_ngo')
+    op.alter_column('events_version', 'show_remaining_tickets',
+               existing_type=sa.BOOLEAN(),
+               nullable=True,
+               autoincrement=False,
+               existing_server_default=sa.text('false'))
```

Comment:
This should not be here

Reply:
@iamareebjamal This is a auto-generated migration.

Reply:
Still